pkg/go/v1/Makefile

```
@@ -176,7 +176,10 @@ test:: $(GENERATED_FILES) $(GO_TEST_REQ)
 # test-until-failure --- Execute all go tests in this module until they fail.
 .PHONY: test-until-failure
 test-until-failure:
-	while go test -count=1 $(GO_TEST_ARGS) ./...; do :; done
+	@counter=0; while go test -count=1 $(GO_TEST_ARGS) ./...; do \
+		counter=$$(expr $$counter + 1); \
+		echo "----------| Test Run $$counter Succeeded |----------"; \
+	done; echo "Test(s) failed after $$counter successful runs"
```

```suggestion
	@counter=0; while go test -count=1 $(GO_TEST_ARGS) ./...; do \
		counter=$$(expr $$counter + 1); \
		echo "--- test run $$counter succeeded"; \
	done; echo "--- failed after $$counter successful run(s)"
```

Just some minor/pedantic suggestions to make the output look more like existing output in the Makefiles.
Assuming it's been tested, LGTM :+1:

---

Infact, my suggestion of having that last `echo` line is probably redundant since the counter is already printed *after* the tests.

---

Has been tested in scenarios including first run fails and 3rd run fails (by manually breaking code and saving in between runs).

Happy to commit that suggestion to better match, was just trying to use font ligature more :p